bip78/src/sender/mod.rs

```diff
@@ -36,7 +36,7 @@ type InternalResult<T> = Result<T, InternalValidationError>;
 /// Builder for sender-side payjoin parameters
 ///
 /// These parameters define how client wants to handle PayJoin.
-pub(super) struct Params {
+pub struct Params {
```

Comment:
This destroys forward-compatibility and is definitely not acceptable. `#[non_exhaustive]` would work but builders are still more convenient for user code.

Reply:
I'm confused about this comment. That change is only on an intermediate commit. The original is `pub struct Params {` as exists on master right now

Reply:
The fields are public and adding a public field to a public struct is breaking change unless it's `non_exhaustive`.